src/scripts/quests/QuestLineHelper.ts

```diff
@@ -1035,6 +1035,120 @@ class QuestLineHelper {
         App.game.quests.questLines().push(metaGroudonQuestLine);
     }
 
+    //Orre Questlines
+    public static createOrreColosseumQuestLine() {
+        const orreColosseumQuestLine = new QuestLine('Shadows in the Desert', 'Explore Orre and uncover the origin of Shadow Pokémon.', new GymBadgeRequirement(BadgeEnums.Elite_HoennChampion), GameConstants.BulletinBoards.Hoenn);
```

Comment:
Does this need a DevelopmentRequirement?

Reply:
Good call, added that commit